tests/e2e/autoscale/00-install.yaml

```diff
@@ -6,9 +6,9 @@ kind: OpenTelemetryCollector
 metadata:
   name: simplest
 spec:
-  minReplicas: 1
```

Comment:
I would prefer to keep this for now to verify the old config works. Maybe add a comment to update it.